src/types/ethereum.ts

```diff
@@ -80,14 +80,14 @@ export const TokenDetailsERC20 = t.type({
   name: t.string, // 'Carrots',
   contract: t.string, // '0x6e0646b014d99d79f4e875b6723fa8e46becbd15',
   transfers: t.number, // 1,
-  symbol: t.string, // 'CEN',
+  symbol: optional(t.string), // 'CEN',
 }, 'TokenDetailsERC20')
 export type TokenDetailsERC20 = t.TypeOf<typeof TokenDetailsERC20>
 
 export const TokenDetailsERC20Balance = extendCodec(
   TokenDetailsERC20,
   {
-    balance: t.string, // '8503600000000000000'
+    balance: optional(t.string), // '8503600000000000000'
```

Comment:
Symbol and balance was undefined here: 

```
  {
         "type": "ERC20",
         "name": "0xE0B7927c4aF23765Cb51314A0E0521A9645F0E2A",
         "contract": "0xE0B7927c4aF23765Cb51314A0E0521A9645F0E2A",
         "transfers": 118
      }
```  

● BlockbookEthereum › getAddressDetails - [faa.st](http://faa.st/) wallet › succeeds

    TypeError: Invalid type - Expected type string | undefined for (Paginated & AddressDetailsEthereumTxids).tokens.16.symbol, but got: undefined

      127 |       return value as T
      128 |     }
    > 129 |     return assertType(codec, value, ...rest)
          |            ^
      130 |   }
      131 |
      132 |

      ✕ succeeds (2117 ms)

  ● BlockbookEthereum › getAddressDetails - [faa.st](http://faa.st/) wallet › succeeds

    TypeError: Invalid type - Expected type string | undefined for (Paginated & AddressDetailsEthereumTxids).tokens.16.balance, but got: undefined

      127 |       return value as T
      128 |     }
    > 129 |     return assertType(codec, value, ...rest)
          |            ^
      130 |   }
      131 |

Reply:
Project was dissolved. see etherscan screeshot

![Screenshot 2022-03-28 at 23 34 28](https://user-images.githubusercontent.com/87442614/160498168-6ed3a3fc-e2ec-4f66-9316-fb7924d95030.png)